src/bin/usage-params.ts

```diff
@@ -8,12 +8,16 @@ export default yargs
   .usage('Parses and runs a migration script on a Contentful space.\n\nUsage: contentful-migration [args] <path-to-script-file>\n\nScript: path to a migration script.')
   .demandCommand(1, 'Please provide the file containing the migration script.')
   .check((args) => {
-    const filePath = path.resolve(process.cwd(), args._[0])
-    if (fs.existsSync(filePath)) {
-      args.filePath = filePath
-      return true
+    if (typeof args._[0] === 'string') {
+      const filePath = path.resolve(process.cwd(), args._[0])
```

Comment:
wouldn't `args._[0].toString()` be sufficient?

Reply:
👍🏻 :)